analyzers/dataframe/FCCAnalyses.cc

```diff
@@ -220,6 +220,16 @@ int get_njets2(ROOT::VecOps::RVec<fcc::JetData> x, ROOT::VecOps::RVec<fcc::JetDa
   return result;
 }
 
+float get_flat(ROOT::VecOps::RVec<float> x, int i)
```

Comment:
I think this doesn't work, no? Could be removed

Reply:
yes, true, this does not work and from to dump into histogram this is not needed. will remove